backend/danswer/file_processing/extract_file_text.py

```diff
@@ -251,47 +262,41 @@ def epub_to_text(file: IO[Any]) -> str:
 
 
 def file_io_to_text(file: IO[Any]) -> str:
-    encoding = detect_encoding(file)
-    file_content_raw, _ = read_text_file(file, encoding=encoding)
-    return file_content_raw
+    try:
```

Comment:
This try should not be here, it can be a try except in the Confluence connector instead. It should still fail loudly in other flows. For example if someone uploads a wrong file type, it should not just be as if they uploaded nothing. It should fail